Doc/using/cmdline.rst

```diff
@@ -1281,6 +1281,14 @@ conflict.
 
    .. versionadded:: 3.14
 
+.. envvar:: PYTHON_JIT
+
+   On builds where experimental just-in-time compilation is available, this
+   variable can force the JIT to be disabled (``0``) or enabled (``1``) at
+   interpreter startup.
+
+   .. versionadded:: 3.13
```

Comment:
We should backport this, should it be split out into a different PR for ease?

Reply:
Is it worth using miss-islington for the backport here?

Reply:
I'll just do it manually in a bit.